run.py

```diff
@@ -452,8 +452,9 @@ def fetch_test_details(var) -> dict:
                 res.append(tmp)
             total_time = "0s"
             send_to_cephci = post_results or post_to_report_portal
-            email_results(res, run_id, trigger_user, run_dir, total_time, send_to_cephci)
-            return
+            info = f"Exception hit during node creation.Traceback: {traceback.format_exc(limit=2)}"
+            email_results(res, run_id, trigger_user, run_dir, total_time, info, send_to_cephci)
```

Comment:
would suggest to apply *args, **kwargs if it is helpful.

Reply:
It is a good direction. IMO, instead of keeping it generic, we could use `test_results` or any other variable name of type `dict`. This would allow us to add/modify without changing the method signature.

Reply:
Added a dictionary to pass various variables to the email alert method.